Tests/Rules/AvoidAssignmentToAutomaticVariable.tests.ps1

```diff
@@ -7,60 +7,61 @@ BeforeAll {
 
 Describe "AvoidAssignmentToAutomaticVariables" {
     Context "ReadOnly Variables" {
+        BeforeDiscovery {
```

Comment:
Added per best practice in Pester v5. Only cosmetic and more explicit in this case.